modules/administration/pages/content-lifecycle.adoc

```diff
@@ -37,49 +35,56 @@ The [guimenu]``Name`` field only accepts lowercase letters, numbers, periods (``
 You can select the order of the environments in the lifecycle by selecting an environment in the [guimenu]``Insert before`` field when you create it.
 
 [NOTE]
-.Content Filters
 ====
-Content Filters are global in your organization and can be shared in multiple projects.
+Content filters are global in your organization and can be shared between projects.
 ====
 
 [NOTE]
-.Adding environment in existing project
 ====
-When you add an environment to an existing project that has already some sources built, the newly added environment will be populated with the content of its predecessor environment.
+If your project already contains built sources, when you add an environment it will automatically populate with the existing content.
+Content will be drawn from the previous environment of the cycle if it had one.
+If there is no previous environment, it will use the latest available built version.
```

Comment:
And I just realized this "part of the feature" is not yet implemented at the moment. It will be, but not right now. We need to remove the case when `there is no previous environment`, or we need to clarify rewording it in `If there is no previous environment, it will be left empty, waiting for a new Build of the project sources."